src/python/pants/backend/docker/registries.py

```diff
@@ -52,13 +51,13 @@ def from_dict(cls, alias: str, d: dict[str, Any]) -> DockerRegistryOptions:
         return cls(
             alias=alias,
             address=d["address"],
-            default=Parser.ensure_bool(d.get("default", alias == "default")),
-            skip_push=Parser.ensure_bool(d.get("skip_push", DockerRegistryOptions.skip_push)),
+            default=d.get("default", alias == "default"),
+            skip_push=d.get("skip_push", DockerRegistryOptions.skip_push),
             extra_image_tags=tuple(
                 d.get("extra_image_tags", DockerRegistryOptions.extra_image_tags)
             ),
-            repository=Parser.to_value_type(d.get("repository"), str, None),
```

Comment:
This was particularly unnecessary, since all that function does when you pass `str` as the type is cast...

Reply:
🤦🏽